caffe2/onnx/helper.cc

```diff
@@ -44,4 +44,27 @@ void DummyName::Reset(const std::unordered_set<std::string> &used_names) {
   counter_ = 0;
 }
 
+NodeProto MakeNode(
+    const std::string& type,
+    const std::vector<std::string>& inputs,
+    const std::vector<std::string>& outputs,
+    const std::vector<AttributeProto>& attributes,
+    const std::string& name) {
+  NodeProto node;
+  if (!name.empty()) {
+    node.set_name(name);
+  }
+  node.set_op_type(type);
+  for (const auto& input: inputs) {
+    node.add_input(input);
+  }
+  for (const auto& output: outputs) {
+    node.add_output(output);
+  }
+  for (const auto& attr: attributes) {
+    auto* tmp = node.add_attribute();
+    tmp->CopyFrom(attr);
```

Comment:
Why do you need the temporary? If you want to check it was successfully created, you can do e.g.
```c++
auto* new_node = node.add_attribute();
CHECK_NOTNULL(new_node);
new_node->CopyFrom(attr);
```
Otherwise, you can just do one-liner `node.add_attribute()->CopyFrom(attr);`

Reply:
I'll do one liner.